native/cocos/renderer/pipeline/custom/RenderInterfaceTypes.h

```diff
@@ -500,6 +1276,9 @@ class SceneVisitor {
     virtual void draw(const gfx::DrawInfo &info) = 0;
 };
 
+/**
+ * @deprecated @en Not used @zh 未使用
+ */
 class SceneTask {
```

Comment:
Can just remote it, as it is not used outside.

Reply:
Currently web-pipeline implementation relies on them. They will be removed in the future.

Reply:
Got it.